test-network-k8s/scripts/prereqs.sh

```diff
@@ -47,7 +47,7 @@ function check_prereqs() {
   if [[ $? -ne 0 ]]; then
     echo "Downloading LATEST Fabric binaries and config"
     curl -sSL https://raw.githubusercontent.com/hyperledger/fabric/main/scripts/bootstrap.sh \
-      | bash -s -- -s -d
+      | bash -s -- ${FABRIC_VERSION} ${FABRIC_CA_VERSION} -s -d
```

Comment:
Good catch.
However, isn't FABRIC_VERSION and FABRIC_CA_VERSION a two digit number in these scripts (e.g. 2.5), whereas the bootstrap.sh script requires a three digit version for pulling binaries (e.g. 2.5.7).

Reply:
Actually in my local copy of network script, I am using 3 digit versions. However I can define a pair of auxiliary variables in order to save both 2 digit and 3 digit version's formats in network script.

Reply:
Would it be enough to make the defaults 3 digit versions? Alternatively, it might be nice if the fabric install script accepted 2 digit versions and "did the right thing" ...and it might be good to use the [newer install script](https://github.com/hyperledger/fabric/blob/main/scripts/install-fabric.sh) instead of bootstrap.sh